style.css

```diff
@@ -0,0 +1,170 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #2A2DD2;
+  color: white;
+  font-size: 100%;
+  line-height: 1.5;
```

Comment:
Change line height to 
`line-height: 1.125;`

Reply:
`1.1875` may work better

Reply:
yeah. okay.